examples/omni-lock-unisat/README.md

```diff
@@ -0,0 +1,21 @@
+# Lumos Works with UniSat
+
+Nervos maintains a powerful lock
+called [Omni Lock](https://github.com/XuJiandong/docs-bank/blob/master/omni_lock.md) (previously named RC lock), which
+can use MetaMask as a signer. This example will show how to use Lumos to send a transaction using Omni lock and MetaMask
```

Comment:
The spelling of `Omni Lock` should be unified, it's called `Omni Lock` at line 4 but `Omni lock` at this line.

BTW, it's called `Omnilock` in the RFC at https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0042-omnilock/0042-omnilock.md